Editor/Scripts/BotManagement/RGBotSynchronizer.cs

```diff
@@ -390,33 +426,16 @@ private string ZipLocalBot(long botId, string sourcePath)
             return md5;
         }
 
-        private void CreateParentFolders()
-        {
-            if (!AssetDatabase.IsValidFolder("Assets/RegressionGames"))
-            {
-                AssetDatabase.CreateFolder("Assets", "RegressionGames");
-            }
-
-            if (!AssetDatabase.IsValidFolder("Assets/RegressionGames/Runtime"))
-            {
-                AssetDatabase.CreateFolder("Assets/RegressionGames", "Runtime");
-            }
-
-            if (!AssetDatabase.IsValidFolder(BOTS_PATH))
-            {
-                AssetDatabase.CreateFolder("Assets/RegressionGames/Runtime", "Bots");
-            }
-        }
-
         /**
-         * <summary>Creates a folder and any needed parent folders for the given botname/botId pair.</summary>
+         * <summary>Creates a folder and any needed parent folders for the given botname in the default bot path.</summary>
          * <param name="replace">If <c>true</c>, any existing content in this directory will be replaced.</param>
          * <returns>String path of the folder</returns>
          */
-        private string CreateBotFolder(string botName, long botId, bool replace)
+        private string CreateDefaultBotFolder(string botName, bool replace)
         {
-            CreateParentFolders();
-            var botFolderName = $"{botName}_{botId}".Replace('-', 'n');
+            RGEditorUtils.CreateAllAssetFolders(BOTS_PATH);
+
+            var botFolderName = $"{botName}".Replace('-', 'n');
```

Comment:
Is this needed anymore since the bot names won't include the ID? This was only to handle negative ID numbers

Reply:
👍 whenever you fix it, feel free to merge!